modules/manage/partials/mountable-topics.adoc

```diff
@@ -0,0 +1,211 @@
+For topics with Tiered Storage enabled, you can unmount a topic to detach segment data that is still on disk to object storage, and unmount a topic from object storage to attach the topic data to either the same origin cluster, or a different one. This allows you to hibernate a topic and free up and reclaim system resources taken up by the topic, or migrate a topic to a different cluster.
+
+Redpanda also transfers topic manifests when mounting or unmounting a topic, so the topic can quickly accept reads and writes again and you can resume cluster workloads with ease. 
+
+An unmounted topic in object storage is detached from all clusters. The original cluster releases ownership of the topic. You can mount a topic from object storage that has been previously unmounted from a source cluster to a different destination cluster, or back to the same source cluster.
+
+== Prerequisites
+
+. xref:get-started:rpk-install.adoc[Install `rpk`], or ensure that you have access to the Admin API.
+. Enable xref:manage:tiered-storage.adoc[Tiered Storage] for specific topics, or for the entire cluster (all topics).
+
+== Unmount a topic from a cluster to object storage
+
+When you unmount a topic, all incoming writes to the topic are blocked as Redpanda unmounts the topic from the cluster to object storage. Producers and consumers of the topic receive an error message indicating that the topic is no longer available. The unmounted topic is deleted in the source cluster.
```

Comment:
"The unmounted topic is deleted in the source cluster."
When does that happen? After the command is successfully ran? Is the command async? If yes, can I track its progress? Is it possible that the command halts mid operation? If yes, what do I do recover?

Reply:
Also I believe we should add more emphasis to this part as it's super important. Suggest bold, but INFO admonition could also work